### DISCUSSION TITLE
Nuxt 3 SSR doesn't work

### DISCUSSION THREAD

**Discussion by razmatinyan** [nuxt/nuxt#23106] [Questions] [answered]
Hello Nuxt community. I have an issue with my Nuxt 3 App. It is not rendering is SSR mode. When I look at page in **`"View page source"`** mode, it is not rendered in SSR. In main `"__nuxt"` block I got this `<div id="__nuxt"></div>` . Here is my package.json and nuxt.config.ts files. Does anyone know what is going 

**nuxt.config.ts**
```javascript
export default defineNuxtConfig({
	ssr: true,
	devtools: { enabled: true },
	modules: ['@nuxt/ui', '@nuxtjs/supabase'],
	ui: {
		global: true,
		icons: ['heroicons'],
	},
	colorMode: {
		preference: 'dark',
	},
	css: ['~/assets/fonts.css'],
	alias: {
		assets: '/<rootDir>/assets',
	},
	supabase: {
		redirectOptions: {
			login: '/',
			callback: '/',
			exclude: [],
		},
	},
	runtimeConfig: {
		public: {
			DOMAIN: process.env.DOMAIN ?? 'http://127.0.0.1:3000/',
		},
	},
});

``` 

**package.json**
```javascript
{
  "name": "nuxt-app",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxt/devtools": "latest",
    "@nuxtjs/supabase": "^1.1.0",
    "nuxt": "^3.7.1"
  },
  "dependencies": {
    "@iconify-json/heroicons": "^1.1.12",
    "@nuxt/ui": "^2.8.1",
    "crypto": "^1.0.1"
  }
}

```

**Comment by TheAlexLichter** [comment]
Can you provide a more minimal reproduction which does not need supabase to reproduce the error or at least a more detailed way to reproduce?

**Reply by razmatinyan** [reply]
I will provide it, but in an several hours. 

P.S. I tried to comment out all the content in my `app.vue`, then add some static text and then look at code, and I get the same result: empty `__nuxt`. You can look at repository of that app, until I will provide reproduction: [https://github.com/razmatinyan/nuxt-link-shortener](https://github.com/razmatinyan/nuxt-link-shortener)

**Reply by razmatinyan** [reply]
Sorry for the late answer, here is the reproduction: [https://stackblitz.com/edit/nuxt-shortener-url?file=app.vue,app.config.ts](https://stackblitz.com/edit/nuxt-shortener-url?file=app.vue,app.config.ts). I left there `example.env` file, with my Supabase datas. So please don't steal it :)

P.S. Problem comes from Supabase. When I fully disable the Supabase the website starts to render in SSR correctly, but when I enable it, i get only empy `__nuxt` block.

**Reply by TheAlexLichter** [reply, marked as answer]
Did you try turning off the redirects via `redirect: false` or to exclude pages via `redirectOptions.exclude`? Trying the former works out and applies SSR as expected.

**Reply by razmatinyan** [reply]
I set the `redirect` to `false` and it fixed my problem with SSR. Thank you!
